tests/transformations/test_clean_imports.py

```diff
@@ -80,9 +80,8 @@ def test_organize_by_sections(self):
 
         import os
 
-        from third_party import Object
-
         from codemodder.codemods.transformations.clean_imports import CleanImports
+        from third_party import Object
```

Comment:
how did this change come about?

Reply:
Sorry, another place I meant to leave a comment. Short answer is: I have no idea. I don't think I touched any code that would have any affect on this. The new version looks reasonable/correct to me, but I'm just not sure.

Reply:
sure! I Don't think we have committed to 3.8 in any official capacity or in documentation so that's good with me.